galgebra/ga.py

```diff
@@ -428,7 +428,7 @@ def __init__(self, bases, **kwargs):
         if self.coords is not None:
             self.coords = list(self.coords)
 
-        self.e = mv.Mv(self.iobj, ga=self)  # Pseudo-scalar for geometric algebra
+        self.e = mv.Mv(self.blades_lst[-1], ga=self)  # Pseudo-scalar for geometric algebra
```

Comment:
Do we have a good replacement for `iobj`? Replace to `blades_lst[-1]` doesn't seem to be fun.

Reply:
Yes, `self.e.obj` is that replacement.

Reply:
Can we use `E()` here?

Reply:
`self.e = self.e` isn't going to work very well ;)

`E()` is implemented as return `self.e` - this is the part where we defined `e` as the multlivector of `self.blades_lst[-1]`

Reply:
Sorry, I was on my phone...

Reply:
Yeah, I was a little paranoid trying to give the raw `self.blades_lst[-1]` a name.